Server/src/models/entities/Dataset.ts

```diff
@@ -13,7 +13,7 @@ import { Accounts } from './Accounts';
  * and has multiple composition references for fuel, oxidizer, and diluent
  */
 @Entity()
-export class Dataset {
+export class Dataset extends BaseEntity {

```

Comment:
just wondering what is purpose of adding the BaseEntity?

Reply:
It allows usage of more functions, like '.save' from the entity itself.

Reply:
i see